Samples/AudioPreprocessing/WinUI/AudioPreprocessing/AudioPreprocessing/MainWindow.xaml.cs

```diff
@@ -0,0 +1,76 @@
+﻿using AudioPreprocessing.Model;
+using AudioPreprocessing.ViewModel;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media.Imaging;
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using Windows.Graphics.Imaging;
+using Windows.Storage;
+using Windows.Storage.Pickers;
+
+// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.
+
+namespace AudioPreprocessing
+{
+    [ComImport, System.Runtime.InteropServices.Guid("3E68D4BD-7135-4D10-8018-9FB6D9F33FA1"), InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+    public interface IInitializeWithWindow
+    {
+        void Initialize([In] IntPtr hwnd);
+    }
+
+    /// <summary>
+    /// Main Window of the app, displays melspectrograms.
+    /// </summary>
+    public sealed partial class MainWindow : Window
+    {
+        [DllImport("user32.dll", ExactSpelling = true, CharSet = CharSet.Auto, PreserveSig = true, SetLastError = false)]
+        public static extern IntPtr GetActiveWindow();
+        public MainWindow()
+        {
+            this.InitializeComponent();
+            this.ViewModel = new PreprocessViewModel();
+        }
+
+        public PreprocessViewModel ViewModel { get; set; }
+        private async void OnOpenClick(object sender, RoutedEventArgs e)
+        {
+
+            // Open a text file.
+            FileOpenPicker openPicker = new FileOpenPicker();
+            openPicker.ViewMode = PickerViewMode.Thumbnail;
```

Comment:
comment says text file but you want wav file right?

Reply:
Changed!